fire/__init__.py

```diff
@@ -12,12 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Python Fire module for third_party."""
+"""The Python Fire module."""
 
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
 
 from fire.core import Fire
+from fire.core import FireExit
 
-__all__ = ['Fire']
+__all__ = ['Fire', 'FireExit']
```

Comment:
Perhaps don't expose FireExit yet until the API for FireTrace stabilizes.
In the mean time, the tests can assertRaises a SystemExit rather than a FireExit.